views/analysis.py

```diff
@@ -349,34 +349,6 @@ def layer_tiles(request):
         return HttpResponseServerError()
 
 
-def layer_thumbnail(request):
```

Comment:
Same reason. No need to create another view for this, because it is redundant.
Because:

1. We only need thumbnail url
2. We can get thumbnail url directly from `analysis.impact_layer`
3. No other part in the code reference this function, so we can safely delete it